.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -148,7 +148,7 @@ private object EditingToolkit : WPComPluginBuild(
 			name = "Run JS tests"
 			scriptContent = """
 				cd apps/editing-toolkit
-				yarn test:js --reporters=default --reporters=jest-teamcity --maxWorkers=${'$'}JEST_MAX_WORKERS
+				yarn test:js --reporters=default --reporters=jest-teamcity --maxWorkers=%JEST_E2E_WORKERS%
```

Comment:
Heads up that I moved the unit tests here from calypso apps to the primary unit test build config in #79097

Reply:
Ah gotcha, I'll take a look. Thanks!

Reply:
Heads up, I'm temporarily reverting https://github.com/Automattic/wp-calypso/pull/79159 because of an issue with the ETK build https://github.com/Automattic/wp-calypso/pull/79159

Reply:
@roo2 Thanks for the heads up. I'll merge my changes in since it should not cause any issues with existing builds, but that means I'd be shifting the conflict resolution burden onto @noahtallen - hope you don't mind.